### DISCUSSION TITLE
Exclude & Only & Whitelist implementation question?

### DISCUSSION THREAD

**Discussion by Mik39907** [ThioJoe/YT-Spammer-Purge#486] [Q&A] [answered]
**Exclude** is working as:
ignore these comments/accounts from report/delete
and **only** should be only report/delete these
Ex:
**only** 7-9, 12, 31, 45, etc...
the ones that i have selected with the numbers(7,8,9,12,31,45) should be now prompted to be deleted/reported?
Because when i do that it asks me if i want to exclude channels i presume it is the rest of the comments/accounts but it is a little counter intuitive 

Possible solution when someone uses **only** it should ignore the rest of the comments directly at least for that run because idk if the whitelist is permanent or temporary for each run?

**Comment by ThioJoe** [comment, marked as answer]
The 'only' command is basically the inverse of the 'exclude' command. Meaning behind the scenes it literally does the same thing as exclude, except on the numbers the user _didn't_ enter. I added a small extra message to clarify in v2.14.3
